src/link_manager/LinkManagerStateMachineActiveStandby.cpp

```diff
@@ -671,10 +671,13 @@ void ActiveStandbyStateMachine::handleMuxStateNotification(mux_state::MuxState::
             MUXLOGWARNING(boost::format("%s: Received unsolicited MUX state change notification!") %
                 mMuxPortConfig.getPortName()
             );
-        }
+        } 
```

Comment:
Remove this extra trailing space please.

Reply:
> Remove this extra trailing space please.

Addressed.